source/documentation/administration_guide/topics/Parameters_for_event_notifications_in_ovirt-engine-notifier.adoc

```diff
@@ -34,9 +34,24 @@ The event notifier configuration file can be found in */usr/share/ovirt-engine/s
 |MAIL_REPLY_TO |none |This variable specifies reply-to addresses in RFC822 format on sent mail, if supported by the mail server.
 |MAIL_SEND_INTERVAL |1 |The number of SMTP messages to be sent for each IDLE_INTERVAL
 |MAIL_RETRIES |4 |The number of times to attempt to send an email before failing.
-|SNMP_MANAGER |none |The IP addresses or fully qualified domain names of machines that will act as the SNMP managers. Entries must be separated by a space and can contain a port number. For example, `manager1.example.com manager2.example.com:164`
-|SNMP_COMMUNITY |public |The default SNMP community.
+|SNMP_MANAGERS |none |The IP addresses or fully qualified domain names of machines that will act as the SNMP managers. Entries must be separated by a space and can contain a port number. For example, `manager1.example.com manager2.example.com:164`
+|SNMP_COMMUNITY |public |The SNMP community.
 |SNMP_OID |1.3.6.1.4.1.2312.13.1.1 |The default trap object identifiers for alerts. All trap types are sent, appended with event information, to the SNMP manager when this OID is defined. Note that changing the default trap prevents generated traps from complying with the {engine-name}'s management information base.
+|SNMP_VERSION |2 |Defines which version of SNMP to use. SNMP version 2 and version 3 traps are supported. Possible values: `2` or `3`.
+|SNMP_ENGINE_ID |none |(SNMPv3) The {engine-name} ID used for SNMPv3 traps. ???How do I get this ID???
```

Comment:
This ID comes from the switch/device to be connected through SNMP

Reply:
Fixed.